src/main/java/edu/cornell/kfs/concur/eventnotification/rest/xmlObjects/ConcurEventNotificationDTO.java

```diff
@@ -1,4 +1,4 @@
-package edu.cornell.kfs.concur.rest.xmlObjects;
+package edu.cornell.kfs.concur.eventnotification.rest.xmlObjects;
```

Comment:
Please see package-info.java in this package.  I needed to add those annotations to get the event notification POJOs to unmarshal correctly.  Since I only wanted those annotations to apply to the event notification POJOs, I moved them into a seperate folder.  I think that is the easiest way to make sure the package-info.java annotations only apply to these POJOs.

Reply:
is this related to the java 11 upgrade? or a different jar version update? Thank you

Reply:
I think this was related to the Java 11 update requiring us to use import XML processing libraries instead of it core Java including them: https://www.dariawan.com/tutorials/java/using-jaxb-java-11/

Reply:
This explains the need for a separate library but not the use of the package info

Reply:
I and Jay talked about this and he explained how he has fixed this by adding the package info and moving the needed classes.